Configurable exit condition constants (#274)
planetarium/NineChronicles.Headless: README.md

  --aws-region <String>                                    The AWS region for AWS CloudWatch (e.g., us-east-1, ap-northeast-2). (Default: )
  --authorized-miner                                       Run as an authorized miner, which mines only blocks that should be authorized.
  --tx-life-time <Int32>                                   The lifetime of each transaction, which uses minute as its unit.  60 (m) by default. (Default: 60)
  --message-timeout <Int32>                                The grace period for new messages, which uses second as its unit.  60 (s) by default. (Default: 60)
  --tip-timeout <Int32>                                    The grace period for tip update, which uses second as its unit.  60 (s) by default. (Default: 60)

>   --demand-buffer <Int32>                                  A number that determines how far behind the demand the tip of the chain will generate an error.  1150 blocks by default. (Default: 1150)

Reviewer: ```suggestion
  --demand-buffer <Int32>                                  A number that determines how far behind the demand the tip of the chain will publish `NodeException` to GraphQL subscriptions. 1150 blocks by default. (Default: 1150)
```

-  `--dev`: Flag to turn on the dev mode.
-  `--dev.block-interval`: Specifies the time interval between blocks by milliseconds in dev mode.
-  `--dev.reorg-interval`: Specifies the size of reorg interval in dev mode.

> -  `--message-timeout`: Specifies the time limit that determines how old the latest message is received will generate an error.

Reviewer: ```suggestion
-  `--message-timeout`: Specifies the time limit that determines how old the latest message is received will publish `NodeException` to GraphQL subscriptions.
```

> -  `--tip-timeout`: Specifies the time limit that determines how old the blockchain's tip is updated will generate an error.

Reviewer: ```suggestion
-  `--tip-timeout`: Specifies the time limit that determines how old the blockchain's tip is updated will  publish `NodeException` to GraphQL subscriptions
```

> -  `--demand-buffer`: Specifies the number that determines how far behind the demand the tip of the chain will generate an error.

Reviewer: ```suggestion
-  `--demand-buffer`: Specifies the number that determines how far behind the demand the tip of the chain will publish `NodeException` to GraphQL subscriptions.
```
